### ISSUE TITLE
Allow for multiple queries to be active

### ISSUE THREAD

**Issue by wtimme** [wtimme/OSM-Completionist#21]
> As a mapper, I want to be able to have multiple queries active at the same time, so that I can add more than one type of information without having to switch the query manually

# Acceptance criterias

- The user can manage a list of queries
- Queries can be enabled/disabled individually
- Multiple queries can be active at the same time

**Comment by tordans** [comment]
I think, other thinks that are useful when building this are 

- A query can have a **custom name** to show in the list
At first this could be the query itself "natural=tree and ref is null" but "Trees without ref-tag" would be easier to read.

- Once I have multiple queries, I need a way to see in the map view, **which query highlights an object**. Especially, since multiple queries can highlight the same object.
Brainstorming: 
  - Each query could get a color (automatically?/selectable?). The highlighter uses this color for the border. In case one object is highlighted by multiple queries, the color-rings could create a kind of rainbow of multiple rings.
  - Also, the object-tooltip that shows on tab could list the queries
     > Tree
     > 🔵 Tree without ref
     > 🔴 Tree without leavecycle
  - Also, the details-layer of an object could list the queries (maybe even with a link to edit/toggle the query) 
     > This tree is recognised by the following overpass queries: …

**Comment by wtimme** [comment]
Thanks for your input!

> A query can have a custom name to show in the list

Since at the moment, [quests from StreetComplete][1] are used, I decided to (for now) go with the question, e. g. "Does this bench have a backrest?". That already allows for an easier identification of the quests.
Later, when users are able to add/edit their own quests, this is just a label and they can choose to name it the way they like.

I've submitted another TestFlight build. Let me know what you think.

[1]: https://wiki.openstreetmap.org/wiki/StreetComplete/Quests
